src/network/http_server.rs

```diff
@@ -295,6 +295,29 @@ pub struct ShardInfo {
     pub mempool_size: u64,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct GetFidsRequest {
+    #[serde(rename = "shardId")]
```

Comment:
Let's remove the rename here and in subsequent fields of a `Request`. We recently went through and standardized the http query params to be snake_case.

Reply:
Done!